internal/db/migrate.go

```diff
@@ -28,6 +28,15 @@ func MigrateSteps(connStr string, migrationsDir string, steps int, logger *logru
 	return finish(m, m.Steps(steps), logger)
 }
 
+func MigrateDown(connStr string, migrationsDir string, logger *logrus.Logger) error {
+	m, err := prepare(connStr, migrationsDir, logger)
+	defer cleanup(m, logger)
+	if err != nil {
+		return err
+	}
+	return finishDown(m, m.Down(), logger)
+}
+
```

Comment:
So I'm not sure how I feel about adding support for something that we'll never do in a deployment. I added the downward migration files for completeness but maybe we should just remove them. Broken migrations should really be fixed by another migration.

I think it's better to just drop/create the database in the test then. But other than that I'm happy with splitting up the tests like you did.

Reply:
I have to admit that I kinda like the fact that the DB can migrate down. But if that's pointless, we can remove the .down files and drop the table or database manually at the end of the test. And if one day we need to step down a migration we still can develop the feature when needed.

Reply:
I think that's best. The only way we'll roll back is via restoring a snapshot in aws rds which doesn't include the migration process. Which would of course be the worst case scenario.

Reply:
https://github.com/osbuild/image-builder/pull/258 opened as well; as soon as the db is in some sort of bad state a human needs to investigate.

Reply:
Yep, this makes complete sense. I'll add a commit to remove all of this .down stuff :)